catalog/gpt-spearphishing-experiment.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Researcher finds that GPT-3.5 and GPT-4 can be used to effectively generate spearphishing messages at scale
+companies: ["OpenAI"]
+categories: ["Cybercrime", "Deception"]
```

Comment:
I don't think this counts as `Deception` under our definition, right? This example would need to demonstrate some sort of emergent behavior in order for that to be the case. In reality, the researcher pretended to be a cybersec researcher in order to get GPT to write these messages — GPT was just trying to be helpful.

```suggestion
categories: ["Cybercrime"]
```